cli/src/pcluster/config/update_policy.py

```diff
@@ -491,6 +491,21 @@ def condition_checker_login_nodes_stop_policy(_, patch):
     return not patch.cluster.has_running_login_nodes()
 
 
+def condition_checker_login_nodes_pool_stop_policy(change, patch):
+    """Check if login nodes are running in the pool in which update was requested."""
+    pool_name = get_pool_name_from_change_paths(change)
+    return not patch.cluster.has_running_login_nodes(pool_name=pool_name)
+
+
+def get_pool_name_from_change_paths(change):
```

Comment:
Could you include a comment that shows an example path. Similar to what is done for `get_q_from_config`